index.js

```diff
@@ -1,17 +1,54 @@
 class SortedList {
-  constructor() {}
+  constructor(items, length) {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item)
+    this.items.sort((a, b) => {
+      return a-b
+    })
```

Comment:
solo faltaría actualizar el length de this.items, porque está inicializado a 0 en el constructor (y el constructor se ejecuta solo una vez cuando crea una nueva instancia de la clase):
`this.length = this.items.length;`
o 
`this.length`